roles/thelounge/templates/config.js.j2

```diff
@@ -0,0 +1,472 @@
+"use strict";
+module.exports = {
+    // ## Server settings
+    // ### `public`
+    //
+    // When set to `true`, The Lounge starts in public mode. When set to `false`,
+    // it starts in private mode.
+    //
+    // - A **public server** does not require authentication. Anyone can connect
+    //   to IRC networks in this mode. All IRC connections and channel
+    //   scrollbacks are lost when a user leaves the client.
+    // - A **private server** requires users to log in. Their IRC connections are
+    //   kept even when they are not using or logged in to the client. All joined
+    //   channels and scrollbacks are available when they come back.
+    //
+    // This value is set to `false` by default.
+    public: false,
+    // ### `host`
+    //
+    // IP address or hostname for the web server to listen to. For example, set it
+    // to `"127.0.0.1"` to accept connections from localhost only.
+    //
+    // For UNIX domain sockets, use `"unix:/absolute/path/to/file.sock"`.
+    //
+    // This value is set to `undefined` by default to listen on all interfaces.
+    host: {{
+        "undefined" | safe
+        if _thelounge__host is not defined
+        else _thelounge__host | to_json
+    }},
```

Comment:
following with the suggestion to decouple the web frontend, here I would just pass what the user says; that way you can also use a unix socket
```suggestion
    host: {{
      (thelounge__listen_addr | to_json)
      if thelounge__listen_addr else "undefined"
    }},
```